app/src/main/java/com/sun/tino/hottrailers/ui/movie_detail/MovieDetailActivity.java

```diff
@@ -57,6 +57,11 @@ private void initViewModel() {
         mViewModel.initViewModel(this, this);
         mViewModel.setInternetListener(this);
         mViewModel.loadMovieDetail(mMovieId);
+        //Check movie favorite
+        mViewModel.getFavoriteMovie(mMovieId).observe(this, movie -> {
+            if (movie != null) mViewModel.isFavorite.set(true);
```

Comment:
anh có thể viết gọn như là 
```suggestion
            mViewModel.isFavorite.set(movie != null);
```